.github/workflows/test.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       - name: Go Coverage
         uses: gwatts/go-coverage-action@v1.3.0
         with:
-          coverage-threshold: 9.0 # current (2023.08.12)
+          coverage-threshold: 8.9
```

Comment:
this was me due to the issue of coverage percentage inclusion not working as fluidly as it should 🙂

Reply:
No worries, it inspired me to find out how to mock client calls. :)  This resource is now more or less covered with unit tests.

Reply:
man, this is awesome! 🤘🏻